client/seatbelt.lua

```diff
@@ -224,34 +214,33 @@ end)
 -- Events
 
 RegisterNetEvent('seatbelt:client:UseHarness', function(ItemData) -- On Item Use (registered server side)
-    local ped = PlayerPedId()
-    local inveh = IsPedInAnyVehicle(ped, false)
-    local class = GetVehicleClass(GetVehiclePedIsUsing(ped))
+    local inveh = IsPedInAnyVehicle(cache.ped, false)
+    local class = GetVehicleClass(GetVehiclePedIsUsing(cache.ped))
```

Comment:
```suggestion
    local class = veh and GetVehicleClass(veh) or 0
```

Reply:
Just this one needs to be changed to `cache.vehicle` instead, that should be better